arb_os/arbowner.mini

```diff
@@ -75,6 +101,144 @@ impure func arbowner_giveOwnership(topFrame: EvmCallFrame, calldata: ByteArray)
     evmOp_return(0, 0);
 }
 
+impure func arbowner_addToReserveFunds(topFrame: EvmCallFrame, calldata: ByteArray) { // () payable
+    if (bytearray_size(calldata) != 4) {
+        evmOp_revert_knownCodePc(address(const::Address_ArbOwner), 50, 0, 0);
+    }
+    let callvalue = evmCallFrame_getCallvalue(topFrame);
+    if (callvalue > 0) {
+        gasAccounting_addToReserveFunds(callvalue);
+    }
+    evmOp_return(0, 0);
+}
+
+impure func arbowner_getFeeRecipient(topFrame: EvmCallFrame, calldata: ByteArray) { // () -> (address)
+    if (bytearray_size(calldata) != 4) {
+        evmOp_revert_knownCodePc(address(const::Address_ArbOwner), 100, 0, 0);
+    }
+
+    if let Some(recipient) = gasAccounting_getFeeRecipient() {
+        let mem = bytearray_set256(bytearray_new(0), 0, uint(recipient));
+        if (evmCallStack_setTopFrameMemory(mem)) {
+            evmOp_return(0, 32);
+        } else {
+            evmOp_revert_knownCodePc(address(const::Address_ArbOwner), 101, 0, 0);
+        }
+    } else {
+        evmOp_revert_knownCodePc(address(const::Address_ArbOwner), 102, 0, 0);
+    }
+}
+
+impure func arbowner_setFeeRecipient(topFrame: EvmCallFrame, calldata: ByteArray) { // (address)
+    if (bytearray_size(calldata) != 36) {
+        evmOp_revert_knownCodePc(address(const::Address_ArbOwner), 110, 0, 0);
+    }
+    let recipient = address(bytearray_get256(calldata, 4));
+    if let Some(_) = gasAccounting_setFeeRecipient(recipient) {
+        evmOp_return(0,0);
+    } else {
+        evmOp_revert_knownCodePc(address(const::Address_ArbOwner), 111, 0, 0);
+    }
+}
+
+impure func arbowner_getFeeRates(topFrame: EvmCallFrame, calldata: ByteArray) { // () -> (uint, uint, uint, uint)
```

Comment:
We should consider making this publicly callable

Reply:
Same as above.